File: RAT/Hades v1.6/Source/xRAT.vb/xRAT/Processes.vb
Imports Microsoft.VisualBasic.CompilerServices
Imports System
Imports System.ComponentModel
Imports System.Diagnostics
Imports System.Drawing
Imports System.Runtime.CompilerServices
Imports System.Threading
Imports System.Windows.Forms

Namespace xRAT
    <DesignerGenerated> _
    Public Class Processes
        Inherits Form
        ' Methods
        Public Sub New()
            Class1.QaIGh5M7cuigS
            AddHandler MyBase.Load, New EventHandler(AddressOf Me.Processes_Load)
            Me.InitializeComponent
        End Sub

        Private Sub Button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Class2.Class4_0.method_6.SendToSelected("PRCLIST")
            Me.Button1.Enabled = False
            Dim num As Integer = 1
            Do
                Thread.Sleep(10)
                Application.DoEvents
                num += 1
            Loop While (num <= 200)
            Me.Button1.Enabled = True
        End Sub

        <DebuggerNonUserCode> _
        Protected Overrides Sub Dispose(ByVal disposing As Boolean)
            Try 
                If (disposing AndAlso (Not Me.icontainer_0 Is Nothing)) Then
                    Me.icontainer_0.Dispose
                End If
            Finally
                MyBase.Dispose(disposing)
            End Try
        End Sub

        <DebuggerStepThrough> _
        Private Sub InitializeComponent()
            Me.icontainer_0 = New Container
            Dim manager As New ComponentResourceManager(GetType(Processes))
            Me.ContextMenuStrip3 = New ContextMenuStrip(Me.icontainer_0)
            Me.KillProcessToolStripMenuItem = New ToolStripMenuItem
            Me.Button1 = New Button
            Me.ListView1 = New ListView
            Me.ColumnHeader1 = New ColumnHeader
            Me.ContextMenuStrip3.SuspendLayout
            Me.SuspendLayout
            Me.ContextMenuStrip3.Items.AddRange(New ToolStripItem() { Me.KillProcessToolStripMenuItem })
            Me.ContextMenuStrip3.Name = "ContextMenuStrip3"
            Dim size As New Size(&H8A, &H1A)
            Me.ContextMenuStrip3.Size = size
            Me.KillProcessToolStripMenuItem.Image = Class5.smethod_6
            Me.KillProcessToolStripMenuItem.Name = "KillProcessToolStripMenuItem"
            size = New Size(&H89, &H16)
            Me.KillProcessToolStripMenuItem.Size = size
            Me.KillProcessToolStripMenuItem.Text = "Kill Process"
            Dim point As New Point(12, &H10D)
            Me.Button1.Location = point
            Me.Button1.Name = "Button1"
            size = New Size(&HB0, &H1B)
            Me.Button1.Size = size
            Me.Button1.TabIndex = 2
            Me.Button1.Text = "Refresh Processes"
            Me.Button1.UseVisualStyleBackColor = True
            Me.ListView1.Columns.AddRange(New ColumnHeader() { Me.ColumnHeader1 })
            Me.ListView1.ContextMenuStrip = Me.ContextMenuStrip3
            Me.ListView1.FullRowSelect = True
            point = New Point(12, 12)
            Me.ListView1.Location = point
            Me.ListView1.MultiSelect = False
            Me.ListView1.Name = "ListView1"
            size = New Size(&HB0, &HFB)
            Me.ListView1.Size = size
            Me.ListView1.Sorting = SortOrder.Ascending
            Me.ListView1.TabIndex = 4
            Me.ListView1.UseCompatibleStateImageBehavior = False
            Me.ListView1.View = View.Details
            Me.ColumnHeader1.Text = "Process Name"
            Me.ColumnHeader1.Width = &HAC
            Dim ef As New SizeF(6!, 13!)
            Me.AutoScaleDimensions = ef
            Me.AutoScaleMode = AutoScaleMode.Font
            size = New Size(200, &H133)
            Me.ClientSize = size
            Me.Controls.Add(Me.ListView1)
            Me.Controls.Add(Me.Button1)
            Me.FormBorderStyle = FormBorderStyle.FixedToolWindow
            Me.Icon = DirectCast(manager.GetObject("$this.Icon"), Icon)
            Me.MaximizeBox = False
            Me.MinimizeBox = False
            Me.Name = "Processes"
            Me.StartPosition = FormStartPosition.CenterScreen
            Me.Text = "Running Processes"
            Me.ContextMenuStrip3.ResumeLayout(False)
            Me.ResumeLayout(False)
        End Sub

        Private Sub KillProcessToolStripMenuItem_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim str As String = Me.ListView1.SelectedItems.Item(0).Text.ToString.Replace(".exe", "")
            Class2.Class4_0.method_6.SendToSelected(("PRCKILL|" & str))
            Dim num As Integer = 1
            Do
                Thread.Sleep(10)
                Application.DoEvents
                num += 1
            Loop While (num <= 200)
            Me.Button1.PerformClick
        End Sub

        Private Sub method_0(ByVal sender As Object, ByVal e As EventArgs)
        End Sub

        Private Sub method_1(ByVal sender As Object, ByVal e As EventArgs)
        End Sub

        Private Sub Processes_Load(ByVal sender As Object, ByVal e As EventArgs)
            Class2.Class4_0.method_6.SendToSelected("PRCLIST")
        End Sub


        ' Properties
        Friend Overridable Property Button1 As Button
            Get
                Return Me._Button1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As Button)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.Button1_Click)
                If (Not Me._Button1 Is Nothing) Then
                    RemoveHandler Me._Button1.Click, handler
                End If
                Me._Button1 = value
                If (Not Me._Button1 Is Nothing) Then
                    AddHandler Me._Button1.Click, handler
                End If
            End Set
        End Property

        Friend Overridable Property ColumnHeader1 As ColumnHeader
            Get
                Return Me.columnHeader_0
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As ColumnHeader)
                Me.columnHeader_0 = value
            End Set
        End Property

        Friend Overridable Property ContextMenuStrip3 As ContextMenuStrip
            Get
                Return Me._ContextMenuStrip3
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As ContextMenuStrip)
                Me._ContextMenuStrip3 = value
            End Set
        End Property

        Friend Overridable Property KillProcessToolStripMenuItem As ToolStripMenuItem
            Get
                Return Me._KillProcessToolStripMenuItem
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As ToolStripMenuItem)
                Dim handler As EventHandler = New EventHandler(AddressOf Me.KillProcessToolStripMenuItem_Click)
                If (Not Me._KillProcessToolStripMenuItem Is Nothing) Then
                    RemoveHandler Me._KillProcessToolStripMenuItem.Click, handler
                End If
                Me._KillProcessToolStripMenuItem = value
                If (Not Me._KillProcessToolStripMenuItem Is Nothing) Then
                    AddHandler Me._KillProcessToolStripMenuItem.Click, handler
                End If
            End Set
        End Property

        Friend Overridable Property ListView1 As ListView
            Get
                Return Me._ListView1
            End Get
            <MethodImpl(MethodImplOptions.Synchronized)> _
            Set(ByVal value As ListView)
                Me._ListView1 = value
            End Set
        End Property


        ' Fields
        <AccessedThroughProperty("Button1")> _
        Private _Button1 As Button
        <AccessedThroughProperty("ContextMenuStrip3")> _
        Private _ContextMenuStrip3 As ContextMenuStrip
        <AccessedThroughProperty("KillProcessToolStripMenuItem")> _
        Private _KillProcessToolStripMenuItem As ToolStripMenuItem
        <AccessedThroughProperty("ListView1")> _
        Private _ListView1 As ListView
        <AccessedThroughProperty("ColumnHeader1")> _
        Private columnHeader_0 As ColumnHeader
        Private icontainer_0 As IContainer
    End Class
End Namespace

File: RAT/Hades v1.6/Source/xRAT.cs/xRAT/Processes.cs
namespace xRAT
{
    using Microsoft.VisualBasic.CompilerServices;
    using System;
    using System.ComponentModel;
    using System.Diagnostics;
    using System.Drawing;
    using System.Runtime.CompilerServices;
    using System.Threading;
    using System.Windows.Forms;

    [DesignerGenerated]
    public class Processes : Form
    {
        [AccessedThroughProperty("Button1")]
        private Button _Button1;
        [AccessedThroughProperty("ContextMenuStrip3")]
        private ContextMenuStrip _ContextMenuStrip3;
        [AccessedThroughProperty("KillProcessToolStripMenuItem")]
        private ToolStripMenuItem _KillProcessToolStripMenuItem;
        [AccessedThroughProperty("ListView1")]
        private ListView _ListView1;
        [AccessedThroughProperty("ColumnHeader1")]
        private ColumnHeader columnHeader_0;
        private IContainer icontainer_0;

        public Processes()
        {
            Class1.QaIGh5M7cuigS();
            base.Load += new EventHandler(this.Processes_Load);
            this.InitializeComponent();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            Class2.Class4_0.method_6().SendToSelected("PRCLIST");
            this.Button1.Enabled = false;
            int num = 1;
            do
            {
                Thread.Sleep(10);
                Application.DoEvents();
                num++;
            }
            while (num <= 200);
            this.Button1.Enabled = true;
        }

        [DebuggerNonUserCode]
        protected override void Dispose(bool disposing)
        {
            try
            {
                if (disposing && (this.icontainer_0 != null))
                {
                    this.icontainer_0.Dispose();
                }
            }
            finally
            {
                base.Dispose(disposing);
            }
        }

        [DebuggerStepThrough]
        private void InitializeComponent()
        {
            this.icontainer_0 = new System.ComponentModel.Container();
            ComponentResourceManager manager = new ComponentResourceManager(typeof(Processes));
            this.ContextMenuStrip3 = new ContextMenuStrip(this.icontainer_0);
            this.KillProcessToolStripMenuItem = new ToolStripMenuItem();
            this.Button1 = new Button();
            this.ListView1 = new ListView();
            this.ColumnHeader1 = new ColumnHeader();
            this.ContextMenuStrip3.SuspendLayout();
            this.SuspendLayout();
            this.ContextMenuStrip3.Items.AddRange(new ToolStripItem[] { this.KillProcessToolStripMenuItem });
            this.ContextMenuStrip3.Name = "ContextMenuStrip3";
            Size size = new Size(0x8a, 0x1a);
            this.ContextMenuStrip3.Size = size;
            this.KillProcessToolStripMenuItem.Image = Class5.smethod_6();
            this.KillProcessToolStripMenuItem.Name = "KillProcessToolStripMenuItem";
            size = new Size(0x89, 0x16);
            this.KillProcessToolStripMenuItem.Size = size;
            this.KillProcessToolStripMenuItem.Text = "Kill Process";
            Point point = new Point(12, 0x10d);
            this.Button1.Location = point;
            this.Button1.Name = "Button1";
            size = new Size(0xb0, 0x1b);
            this.Button1.Size = size;
            this.Button1.TabIndex = 2;
            this.Button1.Text = "Refresh Processes";
            this.Button1.UseVisualStyleBackColor = true;
            this.ListView1.Columns.AddRange(new ColumnHeader[] { this.ColumnHeader1 });
            this.ListView1.ContextMenuStrip = this.ContextMenuStrip3;
            this.ListView1.FullRowSelect = true;
            point = new Point(12, 12);
            this.ListView1.Location = point;
            this.ListView1.MultiSelect = false;
            this.ListView1.Name = "ListView1";
            size = new Size(0xb0, 0xfb);
            this.ListView1.Size = size;
            this.ListView1.Sorting = SortOrder.Ascending;
            this.ListView1.TabIndex = 4;
            this.ListView1.UseCompatibleStateImageBehavior = false;
            this.ListView1.View = View.Details;
            this.ColumnHeader1.Text = "Process Name";
            this.ColumnHeader1.Width = 0xac;
            SizeF ef = new SizeF(6f, 13f);
            this.AutoScaleDimensions = ef;
            this.AutoScaleMode = AutoScaleMode.Font;
            size = new Size(200, 0x133);
            this.ClientSize = size;
            this.Controls.Add(this.ListView1);
            this.Controls.Add(this.Button1);
            this.FormBorderStyle = FormBorderStyle.FixedToolWindow;
            this.Icon = (Icon) manager.GetObject("$this.Icon");
            this.MaximizeBox = false;
            this.MinimizeBox = false;
            this.Name = "Processes";
            this.StartPosition = FormStartPosition.CenterScreen;
            this.Text = "Running Processes";
            this.ContextMenuStrip3.ResumeLayout(false);
            this.ResumeLayout(false);
        }

        private void KillProcessToolStripMenuItem_Click(object sender, EventArgs e)
        {
            string str = this.ListView1.SelectedItems[0].Text.ToString().Replace(".exe", "");
            Class2.Class4_0.method_6().SendToSelected("PRCKILL|" + str);
            int num = 1;
            do
            {
                Thread.Sleep(10);
                Application.DoEvents();
                num++;
            }
            while (num <= 200);
            this.Button1.PerformClick();
        }

        private void method_0(object sender, EventArgs e)
        {
        }

        private void method_1(object sender, EventArgs e)
        {
        }

        private void Processes_Load(object sender, EventArgs e)
        {
            Class2.Class4_0.method_6().SendToSelected("PRCLIST");
        }

        internal virtual Button Button1
        {
            get
            {
                return this._Button1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.Button1_Click);
                if (this._Button1 != null)
                {
                    this._Button1.Click -= handler;
                }
                this._Button1 = value;
                if (this._Button1 != null)
                {
                    this._Button1.Click += handler;
                }
            }
        }

        internal virtual ColumnHeader ColumnHeader1
        {
            get
            {
                return this.columnHeader_0;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this.columnHeader_0 = value;
            }
        }

        internal virtual ContextMenuStrip ContextMenuStrip3
        {
            get
            {
                return this._ContextMenuStrip3;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._ContextMenuStrip3 = value;
            }
        }

        internal virtual ToolStripMenuItem KillProcessToolStripMenuItem
        {
            get
            {
                return this._KillProcessToolStripMenuItem;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                EventHandler handler = new EventHandler(this.KillProcessToolStripMenuItem_Click);
                if (this._KillProcessToolStripMenuItem != null)
                {
                    this._KillProcessToolStripMenuItem.Click -= handler;
                }
                this._KillProcessToolStripMenuItem = value;
                if (this._KillProcessToolStripMenuItem != null)
                {
                    this._KillProcessToolStripMenuItem.Click += handler;
                }
            }
        }

        internal virtual ListView ListView1
        {
            get
            {
                return this._ListView1;
            }
            [MethodImpl(MethodImplOptions.Synchronized)]
            set
            {
                this._ListView1 = value;
            }
        }
    }
}

